impl/src/rendering/templates/partials/features.template

```diff
@@ -0,0 +1,18 @@
+
+    crate_features = [
```

Comment:
Know you're preserving behavior here, but since these lines will be changing in quite a few files, if there are no features, can this not be serialized?